### PR TITLE
add defaultAction for CarouselColumnTemplateBuilder

### LINKED ISSUE #331: Add default action for CarouselColumnTemplateBuilder
[Column object for carousel](https://developers.line.biz/en/reference/messaging-api/#column-object-for-carousel)

Carousel column has default action props.
We should add default action for CarouselColumnTemplateBuilder.

### PR DESCRIPTION
**Pull request by hinoguma**
Fixes #331

### CODE REVIEW

File: tests/LINEBot/MessageBuilder/TemplateBuilder/CarouselColumnTemplateBuilderTest.php [hunk 1]
```diff
@@ -100,6 +115,20 @@ public function test()
...
             }
             $imageBackgroundColor = isset($t['param'][4]) ? $t['param'][4] : null;
+            $defaultAction = null;
+            if (isset($t['param'][5])) {
+                switch ($t['param'][5]) {
+                    case 'postback':
+                        $defaultAction = $postbackActionBuilder;
+                        break;
+                    case 'message':
+                        $defaultAction = $messageTemplateActionBuilder;
```
[the remark this answers is not in the document]
Comment (moririnson) [reply tests/LINEBot/MessageBuilder/TemplateBuilder/CarouselColumnTemplateBuilderTest.php:125]: could you change to write uri test only or all actions tests?

File: src/LINEBot/MessageBuilder/TemplateBuilder/CarouselColumnTemplateBuilder.php [hunk 2]
```diff
@@ -42,6 +42,11 @@ class CarouselColumnTemplateBuilder implements TemplateBuilder
     /** @var array */
     private $template;
 
+    /**
+     * @var TemplateActionBuilder
```
Comment (moririnson) [inline src/LINEBot/MessageBuilder/TemplateBuilder/CarouselColumnTemplateBuilder.php:46]: `TemplateActionBuilder|null`

**Review by Vaduz — COMMENTED** [review]
Implementations are looking good

File: src/LINEBot/MessageBuilder/TemplateBuilder/CarouselColumnTemplateBuilder.php (hunk 2 above)
Comment (hinoguma) [reply src/LINEBot/MessageBuilder/TemplateBuilder/CarouselColumnTemplateBuilder.php]: thank you. fixed

File: tests/LINEBot/MessageBuilder/TemplateBuilder/CarouselColumnTemplateBuilderTest.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (hinoguma) [reply tests/LINEBot/MessageBuilder/TemplateBuilder/CarouselColumnTemplateBuilderTest.php:155]: I changed to write all actions tests.

**Review by moririnson — APPROVED** [review]

**Comment by moririnson** [comment]
@hinoguma thanks!

**Pull request merged**